src/cascadia/TerminalApp/App.cpp

```diff
@@ -843,17 +823,26 @@ namespace winrt::TerminalApp::implementation
     // - tabViewItem: the TabViewItem in the TabView that is being removed.
     void App::_RemoveTabViewItem(const IInspectable& tabViewItem)
     {
+        // TODO: GitHub:627: Need a better story for what should happen when the last tab is closed.
+        if (_tabs.size() <= 1) {

```

Comment:
nit: Our coding standard uses [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) braces, with braces on the line following the conditional.

Reply:
Sorry about it. I will change.